plugins/json_logs.yaml

```diff
@@ -0,0 +1,50 @@
+version: 0.0.1
+title: JSON
+description: Log parser for JSON
+parameters:
+  - name: log_paths
+    type: "[]string"
```

Comment:
this parameter should have `required: true`.

Reply:
I just added that in, thanks!